### PR TITLE
API Endpoints & Database Seeding README Updates

### LINKED ISSUE #9: Add setup instructions to eth_crowdfund_be README
Add the instructions for installing the application and setting up the database to the README for the eth-crowdfund_be app.

### LINKED ISSUE #14: API Endpoint Documentation
- [x] Add Endpoint documentation for Campaigns
- [x] Add Endpoint documentation for Requests

### PR DESCRIPTION
**Pull request by AntonioJacksonII**
# What's this PR do?
Updates the README to include instructions for seeding the database and the API endpoints for Campaigns and Requests

# Where should the reviewer start?
`eth_crowdfund_be/README.md`

# How should this be manually tested?
The reviewer can follow the Seeding the Database instructions to verify if their database properly seeds. The reviewer can follow the instructions for the API endpoints to verify if they are functioning properly.

# Background context?
The FE team should now be able to utilize these endpoints to interact with their local databases. The URIs will need to be updated after this application is deployed to Heroku to reflect the Heroku application URL.

Closes #14 
Closes #9

### CODE REVIEW

**Review by perryr16 — APPROVED** [review]
Nice formatting and documentation

File: eth_crowdfund_be/README.md [hunk 1]
```diff
@@ -4,52 +4,51 @@
...
-        * Debug mode: on
-        * Running on http://127.0.0.1:3000/ (Press CTRL+C to quit)
-        * Restarting with stat
+5. Fork and clone down the repository
+6. Change into the directory `$ cd eth_crowdfund_be/eth_crowdfund_api`
+7. You will need to work in a virtual environment. Why? Using a virtual environment for Python projects allows us to have an isolated working copy of Python so we can work on a specific project without worrying about affecting other projects.
+8. Within the `/eth_crowdfund_api` directory:  
+ a. Run `# pipenv --three` to create the virtual environment  
+ b. Run `$ pipenv shell` to activate the project virtual environment. When you are done working on the project, you should execute `$ exit` to exit the virtual environment  
+ c. Run `$ pipenv install flask flask-sqlalchemy psycopg2 flask-migrate flask-script marshmallow flask-bcrypt pyjwt` to install all dependencies
```
Comment (takeller) [inline eth_crowdfund_be/README.md:28]: I'm not sure if we will end up using bcrypt or pyjwt, so we may be able to remove

File: eth_crowdfund_be/README.md [hunk 2]
```diff
@@ -82,6 +82,14 @@ Path: `eth_crowdfund_be/eth_crowdfund_api`
...
  and you should see `campaigns` and `requests` tables listed.
  1. To exit the database, `$ exit`
+ 
+ ### Seeding the Database
+ Within the `pipenv shell` virtual environment, do the following:
+ 1. Ensure you have the Click package installed by running `$ pipenv install Click`
+ 2. Ensure you are at `/eth_crowdfund_be/eth_crowdfund_api`
+ 3. Run `$ python3 run.py --seed=True`
+ 4. Execute `$ Ctrl + C` to shut down the server and then restart the server with `$ python3 run.py`
+ 5. Visit `localhost:3000/api/v1/campaigns` and `localhost:3000/api/v1/requests` and you should see seeded Campaigns and Requests.
```
Comment (takeller) [inline eth_crowdfund_be/README.md:92]: I don't think you need to restart the server to see the seeds

**Review by takeller — APPROVED** [review]
This all looks great! Thanks for getting this doc set up

File: eth_crowdfund_be/README.md (hunk 2 above)
Comment (AntonioJacksonII) [reply eth_crowdfund_be/README.md:92]: I had an issue with viewing the seeded data initially. Then after I stopped the server, the `Seeding...` and `Finished Seeding` lines appeared. I restarted the server, and then the seeds were there. Did you have to restart the server @perryr16 ?

**Pull request merged**